src/pages/HelpPage.css

```diff
@@ -0,0 +1,23 @@
+.App {
```

Comment:
Adjust names so they match the file name, this will be easier to understand later on